.github/workflows/build-extension.yml

```diff
@@ -72,19 +72,21 @@ jobs:
       - build
     steps:
       - name: Extract branch name
-        run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
         id: extract_branch
+        run: |
+          echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_ENV
+          echo "BRANCH_NAME=$(echo ${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}} | sed 's/[^a-zA-Z0-9_-]//g')" >> $GITHUB_ENV
```

Comment:
Aren't we doing the same thing twice here? branch and BRANCH_NAME are the same?

Reply:
🤦‍♂️ yeah, thanks